src/Monolog/Formatter/MongoDBFormatter.php

```diff
@@ -105,6 +105,12 @@ protected function formatException(\Throwable $exception, int $nestingLevel)
 
     protected function formatDate(\DateTimeInterface $value, int $nestingLevel): UTCDateTime
     {
-        return new UTCDateTime((string) floor($value->format('U.u') * 1000));
+        $miliseconds = floor($value->format('U.u') * 1000);
+
+        $miliseconds = version_compare(phpversion('mongodb'), '1.1.9', '>')
```

Comment:
Yeah, you're right. Wow, now I'm not even sure whether all this makes sense, because it's gonna be ugly fix, and it is ugly already.